frontend/package.json

```diff
@@ -9,7 +9,9 @@
     "lint:fix": "vue-cli-service lint "
   },
   "dependencies": {
+    "@mdi/font": "^4.5.95",
     "axios": "^0.18.1",
+    "moment": "^2.24.0",
```

Comment:
Please don't go this way :(
We probably can get away just with native Date object, if not - date-fns is decent alternative
More info here ;)
https://github.com/you-dont-need/You-Dont-Need-Momentjs